### PR TITLE
BZ 1588151 - Update Ceph RBD dynamic example

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1588151

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 1]
```diff
@@ -161,132 +116,122 @@ metadata:
...
+  userSecretName: ceph-user-secret <7>
+----
+<1> Ceph monitors, comma delimited. This value is required.
+<2> Ceph client ID that is capable of creating images in the pool. The default is `admin`.
+<3> Secret Name for `adminId`. This value is required. The provided secret must have type `kubernetes.io/rbd`.
+<4> The namespace for `adminSecret`. The default is `default`.
+<5> Ceph RBD pool. The default is `rbd`, but this value is not recommended.
+<6> Ceph client ID used to map the Ceph RBD image. The default is the same as `adminId`.
+<7> The name of the Ceph Secret for `userId` to map the Ceph RBD image. It must exist in the
+same namespace as the PVCs. This is required unless it is set as the default in new projects.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: "This is required unless it is set as the default in new projects." I'd replace "it" with exactly what you're referring to and make this sentence active. "Unless you set the Ceph secret as the default in new projects, you must provide this parameter value."

Should "Secret" be capitalized?

I might make it clear that you explain how to do this later. Or add an xref.

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 2]
```diff
@@ -161,132 +116,122 @@ metadata:
...
-[source,yaml]
+. Create the PVC object definition:
++
+[source,bash]
+----
+$ oc create -f ceph-pvc.yaml
+persistentvolumeclaim "ceph-claim-dynamic" created
+----
++
+.PVC object definition example
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: I'd switch the order here, too.

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 3]
```diff
@@ -161,132 +116,122 @@ metadata:
...
     - ReadWriteOnce
   resources:
     requests:
       storage: 2Gi <2>
 ----
-<1> As mentioned above for PVs, the `accessModes` do not enforce access right, but
-rather act as labels to match a PV to a PVC.
-<2> This claim will look for PVs offering `2Gi` or greater capacity.
-
+<1> As mentioned above for PVs, the `accessModes` do not enforce access rights, but
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: Either remove "As mentioned above for PVs," or rewrite to not use "above."

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 4]
```diff
@@ -161,132 +116,122 @@ metadata:
...
+instead act as labels to match a PV to a PVC.
+<2> This claim looks for PVs offering `2Gi` or greater capacity.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: s/offering/that offer

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 5]
```diff
@@ -161,132 +116,122 @@ metadata:
...
     persistentVolumeClaim:
-      claimName: ceph-claim  <5>
+      claimName: ceph-claim <5>  
 ----
 <1> The name of this pod as displayed by `oc get pod`.
-<2> The image run by this pod. In this case, we are telling `busybox` to sleep.
+<2> The image run by this pod. In this case, `busybox` is set to `sleep`.
 <3> The name of the volume. This name must be the same in both the `containers` and
 `volumes` sections.
 <4> The mount path as seen in the container.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: remove "as seen"

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 6]
```diff
@@ -161,132 +116,122 @@ metadata:
...
+After a minute or so, the pod status changes to `Running`.
 
 [[ceph-rbd-dynamic-example-setting-default-secret]]
-== Setting ceph-user-secret as Default for Projects
+== Setting ceph-user-secret as the default for projects
 
-If you want to make the persistent storage available to every project, you must
-modify the default project template. Read more on
+To make persistent storage available to every project, you must
+modify the default project template. Adding this to your default project template
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: s/this/this <noun> (It's not clear enough what you're talking about.)

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 7]
```diff
@@ -317,6 +262,4 @@ objects:
     kubernetes.io/rbd
 ...
 ----
-<1> Place the key from `ceph-user-secret` here in base64 format. See
- xref:../../install_config/storage_examples/ceph_example.adoc#using-ceph-rbd-creating-the-ceph-secret[Creating
- the Ceph Secret].
+<1> Place the key from `ceph-user-secret` here in base64 format.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: How do we get this secret?

File: _topic_map.yml [hunk 8]
```diff
@@ -449,9 +449,9 @@ Topics:
     File: index
   - Name: Sharing an NFS PV Across Two Pods
     File: shared_storage
-  - Name: Complete Example Using Ceph RBD
+  - Name: Using Ceph RBD for Persistent Storage
```
Comment (kalexand-rh) [inline _topic_map.yml]: The old title is still in master, and you didn't change the title on this PR.

**Review by kalexand-rh — COMMENTED** [review]
I have some questions and comments.

File: install_config/storage_examples/ceph_rbd_dynamic_example.adoc [hunk 9]
```diff
@@ -81,76 +82,37 @@ data:
   key: QVFBOFF2SlZheUJQRVJBQWgvS2cwT1laQUhPQno3akZwekxxdGc9PQ== <1>
 type: kubernetes.io/rbd <2>
 ----
-<1> This base64 key is generated on one of the Ceph *MON* nodes using the `ceph auth
-get-key client.admin | base64` command, then copying the output and pasting it
-as the secret key's value.
-<2> This is required for Ceph RBD to work with dynamic provisioning.
---
+<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph auth
+get-key client.admin | base64 command`, then copying the output and pasting it
```
Comment (jhou1) [inline install_config/storage_examples/ceph_rbd_dynamic_example.adoc]: `ceph auth get-key client.admin | base64` command

**Pull request merged**